docs_source_files/content/grid/grid_3/setting_up_your_own_grid.en.md

```diff
@@ -228,9 +228,9 @@ Failure to protect your Grid could result in one or more of the following occurr
 * You allow third parties to access internal web applications and files
 * You allow third parties to run custom binaries
 
-See this blog post on [Detectify](//labs.detectify.com), which gives a good 
+See this blog post on [Detectify Labs](//labs.detectify.com), which gives a good 
 overview of how a publicly exposed Grid could be misused: 
-[Don't Leave your Grid Wide Open](//labs.detectify.com/2017/10/06/guest-blog-dont-leave-your-grid-wide-open/).
+[Don't Leave your Grid Wide Open](//labs.detectify.com/2017/10/06/guest-blog-dont-leave-your-grid-wide-open/). It was written by [Detectify Crowdsource](//cs.detectify.com) hacker, [Peter Jaric](//twitter.com/peterjaric).
 
```

Comment:
Hi @diemol,

i think its fine, but need your conformation too !

Reply:
@Detectify-jocie why is this needed? We are already attributing the author/company.

Reply:
Hi @diemol!

This is to clarify that the link Detectify Labs does not fully represent Detectify as a company - that is our security blog. Also to give credit to our contributor and also let others aware of what is Detectify Crowdsource - our unique ethical hacker platform - for those interested in joining an ethical hacker network and contribute to a safer Internet. The blog author is not employed by Detectify but is a member our growing Crowdsource community. I would like to make it visible that there's community to join in case someone who comes across this documentation is interested in bug bounties.